doc/nrf/glossary.rst

```diff
@@ -0,0 +1,404 @@
+   A register used to prevent read and write access to all CPU registers and memory-mapped addresses.
+
+API Result
+   Indicates success or failure of an API procedure.
+   In case of failure, a comprehensive error code indicating the cause or reason for the failure is provided.
+
+Application
+   An implementation example that includes a variety of libraries to implement a specific use case.
+
+Application Programming Interface (API)
+   A language and message format used by an application program to communicate with an operating system, application, or other services.
+
+Board
+   In Zephyr and |NCS|, a data structure for describing hardware on the board and its configuration files.
+
+Carrier Wave (CW)
+   A single-frequency electromagnetic wave that can be modulated in amplitude, frequency, or phase to convey information.
+
+Cat-M1
+   LTE-M User Equipment (UE) category with a single RX antenna, specified in 3GPP Release 13.
+
+Cat-NB1
+   Narrowband Internet of Things (NB-IoT) User Equipment (UE) category with 200 kHz UE bandwidth and a single RX antenna, specified in 3GPP Release 13.
+
+Chain of Trust
+   A sequence of properties identifying the trustworthiness of each layer in the system, all the way back to a property in the system referred to as *root of trust* (RoT).
+   A secure system depends on building and maintaining a chain of trust through all the layers in the system.
+   Each step in this chain guarantees that the next step can be trusted to have certain properties because any unauthorized modification of a subsequent step will be detected and the process halted.
+
+Clone
+   A local copy of a remote Git repository obtained with ``git clone``.
+
+Commit
+   A Git commit, including a unique SHA and a commit message.
+
+Commit tag
+   A tag prepended to the first line of the commit message to ease filtering and identification of particular commit types.
+
+Contribution
+   A change to the codebase sent to a remote repository for inclusion.
+
+Development Kit (DK)
+   A hardware development platform used for application development.
+
+Device Firmware Update (DFU)
+   A mechanism for upgrading the firmware of a device.
+
+Devicetree
+   A data structure for describing hardware and its boot-time configuration, including peripherals on a board, boot-time clock frequencies, interrupt lines, etc.
+
+Devicetree Specification (DTS, DTSpec)
+   It "provides a complete boot program to client program interface definition, combined with minimum system requirements that facilitate the development of a wide variety of systems." Devicetree Specification, Release v0.3.
+
+Device Under Test (DUT)
+   A manufactured product undergoing testing.
+
+Docker
+   A set of software tools using OS-level virtualization to create and run applications and their dependencies in self-contained environments called containers.
+
+Docker container
+   A self-contained environment created by running a docker container image in the docker engine.
```

Comment:
`Docker` capitalized? Looks like this is their way of writing on the docs.docker.com...

Reply:
Makes sense, it is a trademark